db/migrations/0002_alter_player_guild_alter_player_race_and_more.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.0.2 on 2024-11-18 16:41
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='player',
+            name='guild',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.DO_NOTHING, related_name='guilds', to='db.guild'),
```

Comment:
The `guild` ForeignKey in the `Player` model still uses `DO_NOTHING` for `on_delete`. If `guild` can be `None`, consider using `SET_NULL` with `null=True` to handle `None` values properly.